File: problems/1236-n-th-tribonacci-number/solution.py
class Solution(object):
    def tribonacci(self, n):
        """
        :type n: int
        :rtype: int
        """
        a=0
        b=1
        c=1
        if n==0:
            return a
        if n==1:
            return b
        if n==2:
            return c        

        for i in range(2,n):
            d=a+b+c
            a=b
            b=c
            c=d
        return d    

        

File: problems/1236-n-th-tribonacci-number/solution.java
class Solution {
    public int tribonacci(int n) {
        int a=0, b=1,c=1;
        int d=0;
        if (n==0)
            return a;
        if (n==1)
            return b;
        if (n==2)
            return c ;       

        for(int i=2;i<n;i++)
            {d=a+b+c;
            a=b;
            b=c;
            c=d;
            }
        return d   ;
    }
}
